docs/guides/qiskit-addons-cutting.mdx

```diff
@@ -0,0 +1,154 @@
+
+### Install from PyPI
+
+The most straightforward way to install the `qiskit-addon-cutting` package is with PyPI:
+```bash
+pip install qiskit-addon-cutting
+```
+
+### Install from source
+
+<details>
+<summary>
+Click here to read how to install this package manually.
+</summary>
+
+To contribute to this package or to install it manually, first clone the repository:
+
+```bash
+git clone git@github.com:Qiskit/qiskit-addon-cutting.git
+```
+and install the package with `pip`. To run the tutorials found in the package repository, install the notebook dependencies as well. Install the `dev` dependencies if you plan on developing in the repository.
+```bash
+pip install tox notebook -e '.[notebook-dependencies,dev]'
+```
+
+</details>
+
+### Use within Docker
+
+The dockerfile included in the addon repository can be used to build a Docker image. The included `compose.yaml` file allows you to use the Docker image with the following commands.
+
+<details>
+<summary>
+Click here to read how to use this package within Docker.
+</summary>
+
+```bash
+git clone git@github.com:Qiskit/qiskit-addon-cutting.git
+cd qiskit-addon-cutting
+docker compose build
+docker compose up
+```
+<Admonition type="note" title="Note">
+    If you are using `podman` and `podman-compose` instead of `docker`, the commands are:
+    ```bash
+    podman machine start
+    podman-compose --podman-pull-args short-name-mode="permissive" build
+    podman-compose up
+    ```
+</Admonition>
+
+Once the container is running, you should see a message similar to:
+```
+notebook_1  |     To access the server, open this file in a browser:
+notebook_1  |         file:///home/$USERNAME/.local/share/jupyter/runtime/jpserver-7-open.html
+notebook_1  |     Or copy and paste one of these URLs:
+notebook_1  |         http://e4a04564eb39:8888/lab?token=00ed70b5342f79f0a970ee9821c271eeffaf760a7dcd36ec
+notebook_1  |      or http://127.0.0.1:8888/lab?token=00ed70b5342f79f0a970ee9821c271eeffaf760a7dcd36ec
+```
+
+The *last* URL in this message will give you access to the Jupyter notebook interface. 
+
+Additionally, the home directory includes a subdirectory named persistent-volume. All work you would like to save should be placed in this directory, as it is the only one that will be saved across different container runs.
+
+
+</details>
+
+## Theoretical background
+
+In the circuit cutting process, there are two types of cuts: a **gate** or "space-like" cut, where a cut goes through a gate operating on two (or more) qubits, and a **wire** or "time-like" cut, which cuts directly through a qubit wire (essentially a single-qubit identity gate that has been cut into two pieces).
+
+![Diagram of gate cutting by taking one larger circuit and cutting it into two smaller ones labeled "A" and "B"](/images/guides/qiskit-addons/circuit-cutting-diagram.avif)
```

Comment:
Could we add a figure caption here that indicates this is an example of cutting gates? (that is, 3 entangling gates such that the circuit can be split into two smaller pieces with fewer qubits)